src/groovy/org/pih/warehouse/importer/CSVUtils.groovy

```diff
@@ -153,6 +153,12 @@ class CSVUtils {
         return new CSVPrinter(new StringBuilder(), CSVFormat.DEFAULT)
     }
 
+    static String findSeparator(String firstLine) {
+        List<String> allowedSeparators = [Constants.CUSTOM_COLUMN_SEPARATOR, Constants.TAB_COLUMN_SEPARATOR]
```

Comment:
According to this, `comma` is not allowed as a separator.

Reply:
I didn't add it here, because in previous logic it was falling back to comma even if the columns size != array size after spliting, but I guess it doesn't matter that much, so I'll add it here

Reply:
Then name it `findCustomSepartor`